.github/ISSUE_TEMPLATE/release.yaml

```diff
@@ -0,0 +1,100 @@
+        required: true
+      - label: Kubernetes
+        required: true
+      - label: Istio
+        required: true
+- type: checkboxes
+  id: release_branch
+  attributes:
+    label: "**For all releases** Create Release Branch"
+    description: For this step we're going to start with the `main` branch to create our release branch (e.g. `release/X.Y.Z`) which will later be submitted as a pull request back to `main`.
+    options: 
+      - label: "ensure that you have up to date copy of `main`: git fetch --all"
+        required: true
+      - label: "create the release branch for the version (e.g. `release/1.3.1`): `git branch -m release/x.y.z`"
+        required: true
+      - label: Make any final adjustments to CHANGELOG.md. Double-check that dates are correct, that link anchors point to the correct header, and that you've included a link to the Github compare link at the end.
+        required: true
+      - label: retrieve the latest license report from FOSSA and save it to LICENSES
+        required: true
+      - label: "ensure base manifest versions use the new version (`config/image/enterprise/kustomization.yaml` and `config/image/oss/kustomization.yaml`) and update manifest files: `make manifests`"
+        required: true
+      - label: "push the branch up to the remote: `git push --set-upstream origin release/x.y.z`"
+        required: true
+- type: checkboxes
+  id: release_pr
+  attributes:
+    label: "**For all releases** Create a Release Pull Request"
+    description: For this step we're going to start with the `main` branch to create our release branch (e.g. `release/X.Y.Z`) which will later be submitted as a pull request back to `main`.
+    options:
+      - label: "ensure that you have up to date copy of `main`: `git fetch --all`"
+        required: true
+      - label: "create the release branch for the version (e.g. `release/1.3.1`): `git branch -m release/x.y.z`"
+        required: true
+      - label: Make any final adjustments to CHANGELOG.md. Double-check that dates are correct, that link anchors point to the correct header, and that you've included a link to the Github compare link at the end.
+        required: true
+      - label: retrieve the latest license report from FOSSA and save it to LICENSES
+        required: true
+      - label: "ensure base manifest versions use the new version (`config/image/enterprise/kustomization.yaml` and `config/image/oss/kustomization.yaml`) and update manifest files: `make manifests`"
+        required: true
+      - label: "push the branch up to the remote: `git push --set-upstream origin release/x.y.z`"
+        required: true
```

Comment:
This is accidentally a copy of the preceding section. It should be the steps in https://github.com/Kong/kubernetes-ingress-controller/blob/main/RELEASE.md#release-pull-request

However, there should be two changes to what's there currently:
- The second-to-last step should be amended to "CI will validate the requested version, build and push an image, and run tests against the image before finally creating a tag and publishing a release. If tests fail, CI will push the image but not the tag or release. Investigate the failure, correct it as needed, and start a new release job."
- The last step should be removed. We haven't done this during releases historically, and we often do not need it because backports are rare, and we end up just creating those `A.B.x` branches only if we need to for a backport.